jjb/dynamic/scale-ci_baseline.yml

```diff
@@ -137,6 +140,19 @@
         description: 'Duration to run'
         name: BASELINE_WORKLOAD_DURATION
         trim: 'false'
+    - string:
+        default: "github.com/redhat-performance/perf-dept.git"
+        description: 'Repository container ssh keys for Ansible and pbench server'
```

Comment:
s/Repository container ssh keys for Ansible and pbench server/Repository for ssh keys/g

Reply:
@jtaleric , the description has been updated.